docs(feedback): re-sending of header on resume

dcsiebe opened the pull request:
Note that it is required to re-send the audio header on resume as it is
quite important.

dcsiebe committed:
docs(feedback): re-sending of header on resume

Note that it is required to re-send the audio header on resume as it is
quite important.

joggienl reviewed (changes requested):
I think #74 is created upon this one. I made some comments in there about this piece, so I leave it as "request changes" until the discussion in #73 is resolved.

dcsiebe commented:
To my understanding the related discussion in #74 doesn't request changes, so what changes are requested here?

joggienl commented:
That was the point of starting discussion in the other PR.

joggienl commented on docs/websocket/feedback.md:
```diff
@@ -123,6 +127,16 @@ id   | `string` | **Required** The id of the feedback to pause.
+
+!!! warning
+    It is required to re-send the audio header upon resume. The audio
+    processing will fail if the header isn't re-sent.
```
Same comment as https://github.com/itslanguage/itslanguage-docs/pull/74#discussion_r181974515
It's not needed to have this redundant comment. It's already in bold above. Please remove one of the two.

dcsiebe replied:
Done

dcsiebe committed:
docs(feedback): remove redundant warning

Remove the warning that says the header needs to be re-send when
resuming feedback as it's redundant.
